### PR TITLE
Improve field « EVENT DESCRIPTION»

### LINKED ISSUE #2415: Ergonomics improvement of the field « EVENT DESCRIPTION»
### Feature Description

Within the EVENT module, the current SORMAS version includes a text field “Event description” to formalize useful data concerning event description, comprehension and investigation.

### Problem Description

Users are informed of this size limitation only once entry is validated. Number of characters entered can’t be controlled throughout data entry.
Furthermore, as text is displayed on two lines only, users have to play with the lift to complete and read this zone.

### Proposed Change

It would thus be necessary to:
1)	Implement dynamic display of this zone to make it correspond to the text’s size.
2)	Indicate size limitation  by :

- Blocking data entry when the user reaches the maximum capacity of the field. This modification should be first implemented for the event formulary. It could further on be generalized to all fields of this type within SORMAS application ;
- Displaying the number of characters already entered or the number of characters left.

### Additional Information
screenshot 

image

### PR DESCRIPTION
**Pull request by xavier-calland**
Closes #2415

### CODE REVIEW

File: sormas-api/src/main/resources/captions.properties [hunk 1]
```diff
@@ -41,6 +41,7 @@ total=Total
 unknown=Unknown
 creationDate=Creation date
 notAvailableShort=NA
+numberOfCharacters=Number of characters %d / %d
```
Comment (MateStrysewskeSym) [inline sormas-api/src/main/resources/captions.properties:44]: I'd add a colon after "Number of characters"

File: sormas-ui/src/main/java/de/symeda/sormas/ui/utils/TextFieldWithMaxLengthWrapper.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+	private String buildLabelMessage(String text, T textField) {
+		return String.format(I18nProperties.getCaption(Captions.numberOfCharacters), Strings.nullToEmpty(text).length(), textField.getMaxLength());
+	}
+
+	/**
+	 * Set number of rows in textareas to the number of lines of text + 1.
+	 * Min: {@link #MIN_ROWS}, Max: {@link #MAX_ROWS}
+	 */
+	private void ajustRows(T textField, String text) {
```
Comment (MateStrysewskeSym) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/utils/TextFieldWithMaxLengthWrapper.java:87]: Typo: adjustRows

File: sormas-api/src/main/resources/captions.properties (hunk 1 above)
Comment (xavier-calland) [reply sormas-api/src/main/resources/captions.properties:44]: Done

File: sormas-ui/src/main/java/de/symeda/sormas/ui/utils/TextFieldWithMaxLengthWrapper.java (hunk 2 above)
Comment (xavier-calland) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/utils/TextFieldWithMaxLengthWrapper.java:87]: Done

**Comment by MateStrysewskeSym** [comment]
@xavier-calland We actually use TravisCI for this repository, but for some reason it didn't run for your initial pull request. It apparently does now after your changes though.

**Pull request merged**